src/agent/src/message_queue_utils.hpp

```diff
@@ -15,3 +14,7 @@ boost::asio::awaitable<std::string> GetMessagesFromQueue(std::shared_ptr<IMultiT
 void PopMessagesFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue, MessageType messageType);
 
 void PushCommandsToQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue, const std::string& commands);
+
+std::string GetJsonValue(const nlohmann::json& j, const std::string& key);
```

Comment:
As we talk, there should be a function in the library to do this.

Reply:
Done